api-reference/beta/toc.yml

```
@@ -11709,6 +11709,28 @@ items:
               href: api/riskyusers-confirmcompromised.md
             - name: Dismiss
               href: api/riskyusers-dismiss.md
+          - name: Service principal risk detection
+            items:
+            - name: Risk detection
+              href: resources/serviceprincipalriskdetection.md
+            - name: List risk detections
+              href: api/identityprotectionroot-list-serviceprincipalriskdetections.md
+            - name: Get risk detection
```

Your can use only the verb here because only CRUD operations are supported - unlike the next API group in the TOC which also supports actions/functions
```suggestion
            - name: List
              href: api/identityprotectionroot-list-serviceprincipalriskdetections.md
            - name: Get
```